db/schema/transaction.js

```diff
@@ -0,0 +1,24 @@
+const mongoose = require('mongoose');
+const user = require('user.js')
+
+const transactionSchema = new mongoose.Schema({
+    uuid: String,
+    date: String,
+    items: Array,
+    tax: Number,
+    tip: Number,
+    currency: {
+        type:String,
+        default:"USD"
+    },
+    total: Number,
+    creator: user,
+    people: Array,

```

Comment:
Should the type of array for people be specified as well? E.g. `people: [user]`

Reply:
oh yes, I will adjust it 👍